tg-bot/pom.xml

```diff
@@ -7,14 +7,17 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.4.2</version>
-        <relativePath/>
+        <version>2.7.1</version>
+        <relativePath/> <!-- lookup parent from repository -->
     </parent>
 
     <groupId>com.tutrit.tgbot.example</groupId>
     <artifactId>tg-bot</artifactId>
     <version>1.0-SNAPSHOT</version>
+    <name>tg-bot</name>
+    <description>Demo project for Spring Boot</description>
     <properties>
+        <java.version>17</java.version>
         <maven.compiler.source>17</maven.compiler.source>
         <maven.compiler.target>17</maven.compiler.target>
```

Comment:
either `maven.compiler.source` or `java.version`. Both of them simultaneously usually not used.